src/admin/python/lsst/qserv/admin/cli/entrypoint.py

```diff
@@ -318,31 +359,42 @@ def worker_xrootd(
     )
 
 
-@entrypoint.command(help=f"Start as a replication worker node.\n\n{socket_option_description}")
-@instance_id_option(required=True)
+@entrypoint.command(
+    help="Start as a replication worker node.\n\n"
+         "{socket_option_description}\n\n"
```

Comment:
The string `f` (format type) was dropped in the previous line. Was this done on purpose? Would the parameter substitution still work for `{socket_option_description}`?

Reply:
It was done on purpose. This is not an f string, because `socket_option_description` is not a valid variable when this string is being created (which is how an f-string would get the value of the var). Instead, this string can be rendered later using `.format` e.g.
```
s = "I'd like to say {something}"
...
print(s.format(something="hello world")
```